task_sender/cmd/main.go

```diff
@@ -126,7 +126,7 @@ func taskSenderMain(c *cli.Context) error {
 	}
 
 	var verificationKeyFile []byte
-	if provingSystem == common.GnarkPlonkBls12_381 || provingSystem == common.GnarkPlonkBn254 {
+	if provingSystem == common.GnarkPlonkBls12_381 || provingSystem == common.GnarkPlonkBn254 || provingSystem == common.Groth16Bn254 {
```

Comment:
Good for now, but would be nth utility function for clarity.

e.g `needsVerificationKey(provingSystem)`